recipes/vaex-hdf5/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "vaex-hdf5" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "97f64f3654250d25ef4bc442a9fa42826c68739888c3602b6bd54e260e9055b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - vaex-core>=0.1
+    - h5py
+
+test:
+  imports:
+    - vaex.hdf5
+
+about:
+  home: https://www.github.com/maartenbreddels/vaex
+  license: MIT
+  license_family: MIT
```

Comment:
Is there a license file that we can include?

Reply:
👍 
Now needs to wait for [vaex-core](https://github.com/conda-forge/staged-recipes/pull/4083) to be merged